todo_app/test/test_selenium.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import os
+from todo_app.app import create_app
+from threading import Thread
+from selenium import webdriver
+from todo_app.trello_items import create_trello_board,delete_trello_board
+
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    board_id = create_trello_board()
+    #os.environ['TRELLO_BOARD_ID'] = board_id
```

Comment:
Your app creates a new board, but you are still using your "real" environment variables from .env. So your test is using your "real" board and lists. You should uncomment this line, and you will also need to update  `os.environ[TODOID]` and `os.environ['DOING']` to values based on your temporary board.

You can get the lists with 
```
lists = requests.get(f'https://api.trello.com/1/boards/{board_id}/lists', params=...).json()
```
(with the correct key and token in `params`)

And then just use `lists[0]['id']` and `lists[1]['id']`